app/assets/javascripts/static.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Application entry point.
+ */
+require([
+  'jquery',
+  'underscore',
+  'Class',
+  'backbone',
+  'views/AboutView'
+], function($, _, Class, Backbone, AboutView) {
+
+  'use strict';
+
+  var Static = Class.extend({
+
+    $el: $('body'),
+
+    init: function() {
+      this._initViews();
+    },
+
+    /**
+     * Initialize the map by starting the history.
+     */
+    _initApp: function() {
+      if (!Backbone.History.started) {
+        Backbone.history.start({pushState: true});
+      }
+    },
+
+    /**
+     * Initialize Application Views.
+     * CAUTION: Don't change the order of initanciations if
+     * you are not completely sure.
+     */
+    _initViews: function() {
+      new AboutView();
```

Comment:
We have event tracking in other places in the project, but most of them were already tied to other events and we just added the google analytics tracking to the same method. Maybe there should be a higher catch for the event tracking that then bubbled up the event chain. But maybe I won't try to refactor it at this stage :p For download things like this one, this is the first place I believe. I could rename it to Download view, if that makes it more clear. I named it with the name of the page that will use it.